metrics/main.go

```diff
@@ -26,6 +26,9 @@ func main() {
 		opts.Usage()
 		os.Exit(0)
 	}
+	if len(opts.AllowedNamespaces) == 0 {
+		klog.Fatal("Atleast one 'namespaces' arg should be passed. None given.")
+	}
```

Comment:
We default to "openshift-storage" so this validation is not needed.

Reply:
Removed